diff -b -u openresolv-3.9.0-orig/resolvconf.in openresolv-3.9.0/resolvconf.in
--- openresolv-3.9.0-orig/resolvconf.in	2016-12-29 13:55:44.000000000 -0800
+++ openresolv-3.9.0/resolvconf.in	2018-10-21 00:03:12.000000000 -0700
@@ -780,6 +780,7 @@
 a)
 	# Read resolv.conf from stdin
 	resolv="$(cat)"
+  echo $resolv >> /tmp/resolvconf-a.stdin
 	changed=false
 	changedfile=false
 	# If what we are given matches what we have, then do nothing
